images/win/scripts/Installers/Install-GitHub-CLI.ps1

```diff
@@ -3,8 +3,13 @@
 ##  Desc:  Install GitHub CLI
 ################################################################################
 
-Choco-Install -PackageName gh
+Write-Host "Get the latest gh version..."
+
+$Name = "gh_windows_amd64.msi"
+$DownloadUrl = (Invoke-RestMethod -Uri "https://api.github.com/repos/cli/cli/releases/latest").assets.browser_download_url -match "windows_amd64.msi"
```

Comment:
@samcoe , builds failed because `$DownloadUrl = (Invoke-RestMethod -Uri "https://api.github.com/repos/cli/cli/releases/latest").assets.browser_download_url -match "windows_amd64.msi"` line returns the array with 1 element and it can't be converted properly.
I suggest to replace it to 
```
$Assets = (Invoke-RestMethod -Uri "https://api.github.com/repos/cli/cli/releases/latest").assets.browser_download_url
$DownloadUrl = ($assets -match "windows_amd64.msi") | Select-Object -First 1
```